src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -72,7 +78,7 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
 };
 
 const Small: React.FC<ChannelImageBadgeProps> = ({type}) => {
-  if (type === 'COMMUNITY') {
+  if (type === CHANNEL_COMMUNITY) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition check for the type of channel has been updated to use the constant `CHANNEL_COMMUNITY` instead of the hard-coded string 'COMMUNITY'. This is a good practice as it reduces the risk of typos and makes the code easier to maintain.
```diff
- if (type === 'COMMUNITY') {
+ if (type === CHANNEL_COMMUNITY) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->